bandwidth-file-spec.txt

```diff
@@ -118,9 +118,18 @@ It consists of:
 
     [At start, exactly once.]
 
-    The Unix Epoch time in seconds when the file was created.
-    It does not follow the KeyValue format for backwards
-    compatibility with version 1.0.0.
+    The Unix Epoch time in seconds of the most recent generator result.
```

Comment:
I think we should change "generator result" to "generator bandwidth result".
Otherwise it's a bit confusing.